input/resources/CodeSystem-ehealth-task-handover-roles.json

```diff
@@ -0,0 +1,49 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "ehealth-task-handover-roles",
+  "url": "http://ehealth.sundhed.dk/cs/task-handover-roles",
+  "version": "0.1.0",
+  "name": "TaskHandoverRoles",
+  "title": "Task Handover Roles",
+  "status": "active",
+  "experimental": false,
+  "date": "2024-10-21T00:00:00+00:00",
+  "publisher": "ehealth.sundhed.dk",
+  "contact": [
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://ehealth.sundhed.dk/terminology"
+        }
+      ]
+    }
+  ],
+  "description": "Task Handover Roles",
+  "caseSensitive": true,
+  "content": "complete",
+  "concept": [
+    {
+      "code": "handover-recipient",
```

Comment:
Suggest the following roles:

- inviting-entity (entity inviting another entity to partake in involvement and/or responsibility)
- arriving-entity (entity entering involvement and/or responsibility)
- leaving-entity (entity leaving involvement and/or responsibility)
- remaining-entity (entity continuing involvement and/or responsibility)

The typical scenario could be handover with leaving-entity and arriving-entity. Another scenario is inviting another, where there is inviting-entity and arriving-entity.

Reply:
da: inviterende-part, tilgående-part, afgående-part, tilbageværende-part